package.json

```diff
@@ -56,6 +56,7 @@
     "react-native-splash-screen": "^3.1.1",
     "react-native-svg": "^6.5.2",
     "react-native-text-input-mask": "^0.8.0",
+    "react-native-touch-id": "git://github.com/naoufal/react-native-touch-id#d2ff356f5dbed4ad647b26110f5053ed41f522a4",
```

Comment:
why we need to link to the repo? isn't the lib published? please add a story to the backlog in case we will need to update this reference once this commit gets released, thanks

Reply:
Yes the commit i used includes a fix not already released. I have created the story https://www.pivotaltracker.com/story/show/163153883.